app/src/main/kotlin/com/wire/android/notification/broadcastreceivers/CallNotificationDismissReceiver.kt

```diff
@@ -38,14 +38,15 @@ class CallNotificationDismissReceiver : BroadcastReceiver() {  // requires zero
 
         val userId: QualifiedID? = intent.getStringExtra(EXTRA_RECEIVER_USER_ID)?.toQualifiedID(qualifiedIdMapper)
 
+        // TODO: use the CoroutineScope from the user session
```

Comment:
Do we have task for that?

Reply:
I don't think so, but I will add one, and we can discuss it in the collective